### PR TITLE
Added Authentication

### PR DESCRIPTION
**Pull request by nikhilmalviya-dev**
in this page i Added Login and sign up page
login
signUp
.

### CODE REVIEW

**Comment by ganesh-kanchi** [comment]
Build failed because of all the warnings in your deploy preview log which are:

10:59:51 AM: src/Components/ProductListing/ProductListing.jsx
10:59:51 AM:   Line 88:15:   No duplicate props allowed  react/jsx-no-duplicate-props
10:59:51 AM:   Line 100:15:  No duplicate props allowed  react/jsx-no-duplicate-props
10:59:51 AM: src/Components/footer/Footer.jsx
10:59:51 AM:   Line 10:11:  Anchors must have content and the content must be accessible by a screen reader  jsx-a11y/anchor-has-content
10:59:51 AM:   Line 10:31:  No duplicate props allowed   react/jsx-no-duplicate-props
10:59:51 AM:   Line 11:11:  Anchors must have content and the content must be accessible by a screen reader  jsx-a11y/anchor-has-content
10:59:51 AM:   Line 13:13:  No duplicate props allowed react/jsx-no-duplicate-props
10:59:51 AM:   Line 16:11:  Anchors must have content and the content must be accessible by a screen reader  jsx-a11y/anchor-has-content
10:59:51 AM:   Line 18:13:  No duplicate props allowed react/jsx-no-duplicate-props

**Comment by ganesh-kanchi** [comment]
> Build failed because of all the warnings in your deploy preview log which are:
> 
> 10:59:51 AM: src/Components/ProductListing/ProductListing.jsx 10:59:51 AM: Line 88:15: No duplicate props allowed react/jsx-no-duplicate-props 10:59:51 AM: Line 100:15: No duplicate props allowed react/jsx-no-duplicate-props 10:59:51 AM: src/Components/footer/Footer.jsx 10:59:51 AM: Line 10:11: Anchors must have content and the content must be accessible by a screen reader jsx-a11y/anchor-has-content 10:59:51 AM: Line 10:31: No duplicate props allowed react/jsx-no-duplicate-props 10:59:51 AM: Line 11:11: Anchors must have content and the content must be accessible by a screen reader jsx-a11y/anchor-has-content 10:59:51 AM: Line 13:13: No duplicate props allowed react/jsx-no-duplicate-props 10:59:51 AM: Line 16:11: Anchors must have content and the content must be accessible by a screen reader jsx-a11y/anchor-has-content 10:59:51 AM: Line 18:13: No duplicate props allowed react/jsx-no-duplicate-props

fix these and the checks will pass and let you merge.

File: src/routes/authentication-page/login.jsx [hunk 1]
```diff
@@ -1,6 +1,32 @@
...
+        <div class="container">
+        <div class="login">
+            <h1><strong> Login </strong></h1>
+            <label for="email">
+                Email address
+                <input type="text" placeholder="abcd@gmail.com" class="email authLable"></input>
+            </label>
+            <label for="password">
+                Password
+                <input type="password" placeholder="*********" class="password authLable"></input>
```
Comment (mnath30) [inline src/routes/authentication-page/login.jsx]: Change class to className

File: src/routes/authentication-page/login.jsx [hunk 2]
```diff
@@ -1,6 +1,32 @@
...
+            </label>
+            <label for="remember">
```
Comment (mnath30) [inline src/routes/authentication-page/login.jsx:17]: Replace for with htmlFor in react

**Review by mnath30 — CHANGES_REQUESTED** [review]
Code looks clean. Suggested a few changes.

File: src/routes/authentication-page/login.jsx (hunk 1 above)
Comment (nikhilmalviya-dev) [reply src/routes/authentication-page/login.jsx]: ok will do that

**Pull request merged**